tests/fixtures/netex_france/input_ntfs/equipments.txt

```diff
@@ -0,0 +1,3 @@
+equipment_id,wheelchair_boarding
+0,1
```

Comment:
Only for easier reading of the fixtures, how about have the identifiers `equipment_id` named like `eq:1` or a mix of letters and numbers which help identifying them as identifiers... But maybe it's only me who's annoyed by it :wink:

Reply:
I will do this in unit tests. We can't do it in fixtures.
Equipments are created dynamically on gtfs reading (with ids from 0 to n).
And ids are present in the output xml.

Reply:
Ah yes, indeed